test/crud-test.html

```diff
@@ -876,6 +904,45 @@
           btnConfDialog(crud.$.confirmCancel, 'confirm').click();
           expect(crud.editorOpened).to.be.false;
         });
+
+        it('should open same row again after item closed after click on "New"', (done) => {
+          crud.__mobile = false;
+
+          fakeClickOnRow(0);
+          expect(crud.editorOpened).to.be.true;
+          crud.$.new.click();
+          setTimeout(() => {
+            expect(crud.editorOpened).to.be.true;
+            setTimeout(() => {
```

Comment:
I had to add these two nested `setTimeout`s to let all the events propagate and listenned. Removing these would make it not touch the code you pointed out before.

Reply:
Hmm, I would prefer to avoid this asynchronicity in test, but I guess there is no way to do that as we will need to get rid out of timeout for `__clearItemAndKeepEditorOpened`.